<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="IBM-PF3P51VK" failures="1" tests="3" name="Submit Order Test" time="50.209" errors="0" timestamp="2025-04-15T16:51:10 IST">
  <testcase classname="Snaco.SubmitOrderTest" name="SubmitOrder" time="17.051"/>
  <testcase classname="Snaco.SubmitOrderTest" name="SubmitOrder" time="16.678"/>
  <testcase classname="Snaco.SubmitOrderTest" name="OrderHistoryTest" time="0.812">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Snaco.SubmitOrderTest.OrderHistoryTest(SubmitOrderTest.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- Submit Order Test -->
